src/html/en_uiProperties.json

```diff
@@ -433,7 +433,7 @@
 "MSG0428":"{box name}/{collection name}/{service name}",
 "MSG0429":"Special characters such as control codes can not be used.",
 "MSG0430": "External Role edited successfully!",
-"MSG0431": "personium-localunit://{CellName}/",
+"MSG0431": "personium-localunit:/{CellName}/",
```

Comment:
Akio,

We should approve this pull request first. Then change the message again for the latest format ("personium-localunit:dixonsiu:/").